### PR TITLE
fix logging

### PR DESCRIPTION
**Pull request by hjmodi**
# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] Python type annotations added to new code
- [ ] JSON/YAML configuration changes are updated in the relevant schema
- [ ] Changes to metadata also update the documentation for the metadata
- [ ] Pull request has a link to an osbs-docs PR for user documentation updates
- [ ] New feature can be disabled from a configuration file

### CODE REVIEW

**Review by rcerven — APPROVED** [review]

**Comment by rcerven** [comment]
don't forget to remove bumping osbs commit before merging

File: atomic_reactor/util.py [hunk 1]
```diff
@@ -1389,6 +1389,9 @@ def get_log_files(self, osbs: OSBS, pipeline_run_name) -> List[Output]:
             if task_platform in logfiles:
                 log_file = logfiles[task_platform]
             else:
+                logfiles['noarch'].write(bytearray(f'{task_platform} build has started. '
+                                                   + 'Check platform specific logs\n', 'utf-8'))
```
Comment (chmeliik) [inline atomic_reactor/util.py:1393]: Shouldn't we write this only if `task_platform != 'noarch'`?

Comment (rcerven) [reply atomic_reactor/util.py:1393]: mmmm right it should

Comment (hjmodi) [reply atomic_reactor/util.py:1393]: yes, this will only occur for `!= 'noarch'` because `noarch` will already be in `logfiles`

**Comment by rcerven** [comment]
but I don't think we even need this change, because that is used just for koji build logs which are imported

so nobody can be even watching osbs-build.log at this point

**Comment by hjmodi** [comment]
agreed, closing it.

**Comment by hjmodi** [comment]
Don't need it.

**Pull request closed without merging**